### DISCUSSION TITLE
Template Repository Placeholders [Suggestion]

### DISCUSSION THREAD

**Discussion by ericlmao** [community/community#86543] [Repositories]
### Select Topic Area

Product Feedback

### Body

I suggest improving **Template Repositories**. I think it would be helpful to have a feature called "placeholders." The user who is cloning the repository would fill out these placeholders before it is actually cloned. Once filled out, the placeholders would be replaced with the user's specified information. This feature would be handy during the initial setup process of a repository, where specific details, such as the project name or packaging names, must be changed each time.

**Comment by george-gca** [comment]
I would like this also. In my case we have a [template repository](https://github.com/alshedivat/al-folio) for an academic portfolio website, so a lot of our users are not prolific coding people. When one creates a repo from our template, there are a few changes that must be made so it works for them.

It would be nice if some of these changes could be made out-of-the-box when someone creates a repository from this template, or they could fill in this information at the time that their repo is being created from the template, the same way that they select which branches to include, visibility, and so on.
